doc/ref_model/README.rst

```diff
@@ -0,0 +1,69 @@
+Reference Model
+===============
+
+Release Information
+-------------------
+
+**Bundle: 6**
+
+**Version: 6.0**
+
+**Release Date: 29th January 2021**
+
+Bundle/Version History
+----------------------
+
+============== =================== =============================
+Bundle.Version Date                Note
+============== =================== =============================
+1.0            16th July 2019      First Release
+1.1            2nd August 2019     Post Paris F2F clean up
+2.0            20th September 2019 Botrange Release
+3.0-alpha      10th January 2020   Snezka Release
+3.0            15th May 2020       Baldy Release
+4.0            25th Sep 2020       Baraque Release
+5.0            29th Jan 2021       Elbrus Release
+5.1            8th March 2021      Kali Release work started
+6.0            1 July 2021         Kali Release work finished
+6.1            1 July 2021         Lakelse Release work started
+7.0            7 December 2021     Lakelse Release work finished
```

Comment:
The date needs to be changed to January 4, 2022.

Reply:
I would prefer to do that in a separate rst. This date was merged from the master.